docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
So unfortunately the files in `docs/` are not actually the source files and shouldn't be hand edited. The files in `src` are where the `scss` files should be edited which will then be copied into `docs`.

I'm sorry for this confusion, it was done to limit complications with the Jekyll docs site and we're actually working on fixing it right now. In the meantime, it would help us if you could move your code changes in this file to `src/stylesheets/components/_accordions.scss` and then run the build process with `npm run build` or the more specific gulp tasks.

Let me know if you'd like some help with this. You've been a great contributor so I'd be happy to get these files into the correct form.